graphene_mongo/fields.py

```diff
@@ -160,7 +162,15 @@ def connection_resolver(cls, resolver, connection, model, root, info, **args):
         iterable = resolver(root, info, **args)
         if not iterable:
             iterable = cls.get_query(model, info, **args)
-        _len = len(iterable)
```

Comment:
And then we can simplify as follows to get `iterable` and `_len`:
```python
iterable, _len = cls.get_query(model, info, **args)
```

Reply:
I'll have a look at it today
Hopefully I will be more active on this account now